ykcs11/objects.c

```diff
@@ -1437,8 +1438,12 @@ CK_RV check_create_ec_key(CK_ATTRIBUTE_PTR templ, CK_ULONG n, CK_BYTE_PTR id,
     if (ec_params_len != 10 || memcmp(ec_params, PRIME256V1, ec_params_len) != 0)
       return CKR_ATTRIBUTE_VALUE_INVALID;
   }
-  else /*if () TODO: P384*/
-    return CKR_ATTRIBUTE_VALUE_INVALID;
+  else if (*value_len == 48 || *value_len == 47) {
+    if (ec_params_len != 7 || memcmp(ec_params, SECP384R1, ec_params_len) != 0)
+      return CKR_ATTRIBUTE_VALUE_INVALID;
+  }
+  else
```

Comment:
braces please :)

Reply:
I would much rather use braces, but this will make the style inconsistent with the rest of the code. Do you still think it's ok to add the braces?

Reply:
given that the if already has braces, this should have had braces all along.

One rainy day I'll do a pass and fix them all.

Reply:
Fixed